app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -282,23 +207,103 @@ function initializePodcastPlayback() {
     }
   }
 
-  function readableDuration(seconds) {
-    var sec = Math.floor(seconds);
-    var min = Math.floor(sec / 60);
-    min = min >= 10 ? min : '0' + min;
-    sec = Math.floor(sec % 60);
-    sec = sec >= 10 ? sec : '0' + sec;
-    return min + ':' + sec;
+  function updateProgressListener(audio) {
+    return e => {
+      updateProgress(e, audio);
+    };
   }
 
   function terminatePodcastBar(audio) {
+    // eslint-disable-next-line no-restricted-globals
```

Comment:
where does this `event` come from?

Reply:
It's a DOM `event` so I think this is defined globally because is not defined in this file.